python/ray/data/tests/test_actor_pool_map_operator.py

```diff
@@ -155,6 +190,30 @@ def test_return_actor(self):
         assert pool.num_idle_actors() == 1  # Actor should now be idle.
         assert pool.num_free_slots() == 999
 
+    def test_returned_actor_to_running(self):
```

Comment:
this test should no longer be needed, as we've removed the logic of marking the actor as alive upon returning it?